pkg/apis/gcp/validation/shoot_test.go

```diff
@@ -79,3 +144,7 @@ func validateWorkerConfig(workers []core.Worker, workerConfig *gcp.WorkerConfig)
 
 	return allErrs
 }
+
+func getStringPointer(s string) *string {
+	return &s
+}
```

Comment:
You can use something like `pointer.StringPtr()` like we use [here](https://github.com/gardener/gardener-extension-provider-gcp/blob/7074d6c4528d46e1c72e5200ecf65b2ff869c67b/pkg/apis/gcp/validation/infrastructure_test.go#L58).